packages/core-data/src/fetch/__experimental-fetch-link-suggestions.js

```diff
@@ -85,7 +85,11 @@ const fetchLinkSuggestions = async (
 ) => {
 	const {
 		isInitialSuggestions = false,
-		initialSuggestionsSearchOptions = undefined,
+		initialSuggestionsSearchOptions = {
+			type: 'post',
+			subtype: 'page',
+			perPage: 20,
+		},
```

Comment:
This causes a bug where the variations of the Nav Link Block now don't show suggestions which match their type. Needs fixing.

Reply:
Yes, once this is resolved I think this is good to go.

Reply:
@jeryj I set these as defaults but we can't do this. Instead we need to move this to the Rich Text link format under `@wordpress/formats`.

Reply:
✅